app/models/manageiq/providers/vmware/infra_manager/vm.rb

```diff
@@ -5,6 +5,10 @@ class ManageIQ::Providers::Vmware::InfraManager::Vm < ManageIQ::Providers::Infra
   include_concern 'RemoteConsole'
   include_concern 'Reconfigure'
 
+  supports :clone do
+    unsupported_reason_add(:clone, 'Clone operation is not supported') if blank? || orphaned? || archived?
```

Comment:
Should the unsupported message have `_()` around it?  Also what is a `blank?` vm?